spec/views/application_letters/check.html.erb_spec.rb

```diff
@@ -2,23 +2,33 @@
 
 RSpec.describe "application_letters/check", type: :view do
 
-  it "should show an upload form for an agreement letter for profiles with an age of <18" do
+  before(:each) do
+    @application_letter = assign(:application_letter, FactoryGirl.create(:application_letter))
+  end
 
+  it "should show an upload form for an agreement letter for profiles with an age of <18 and accepted application" do
     @user = FactoryGirl.create(:user)
+    @user.profile = FactoryGirl.create(:profile, user: @user)
     event = FactoryGirl.create(:event)
-    @application_letter = FactoryGirl.create(:application_letter_accepted, user: @user, event: event)
-    @application_letter.user.profile = FactoryGirl.build(:profile)
+    @application_letter = FactoryGirl.create(:application_letter, status: :accepted, user: @user, event: event)
```

Comment:
Defining a new factory to set a single attribute isn't good style, since you can only use one factory. So I think application_letter_accepted should never have existed in the first place, since attributes can easily be set using this way. Both work for now. So what is wrong with this?